HW05.Rmd

```diff
@@ -150,10 +169,23 @@ CTRP_df_q3 <- CTRP_df %>%
   summarize(AUC = mean(area_under_curve)) %>%
   arrange(AUC)
 
-CTRP_df_q3
+cpd_labels <- c("Afatinib", "Neratinib", "Gefatinib", "WZ8040", "Canertinib", "Erlotinib:PLX-4032", "Vandetanib", "Erlotinib", "Lapatinib", "WZ4002", "PD 153035") #Simplified compound names
```

Comment:
I don't understand entirely why you made this to change the labels. For the sake of keeping their names clean and simplified? It's just a lot of writing that is hard-coded in. If for some reason you made this plot later and the order of the compounds was changed, you would have to change this vector too